AnnoyancesFilter/sections/cookies_whitelist.txt

```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that fixes incorrect blocking. Rules should be specific.
```

Comment:
Shouldn't it be specified as 'incorrect blocking of cookie notifications'? Might be misleading as it is now.

Reply:
I can also change it, but it's kinda a clear point that the user add fixes to the `cookie_whitelist.txt`.